tests/settings.py

```diff
@@ -19,12 +19,14 @@
 if RECORD_MODE == 'none':
     STORE_NAME = fake_settings.STORE_NAME
     TENANT_ID = fake_settings.TENANT_ID
-    TOKEN = dict(
-        access=str(base64.b64encode(os.urandom(1420))),
-        refresh=str(base64.b64encode(os.urandom(718))),
-        time=time.time(), client='common',
-        resource="https://management.core.windows.net/",
-        tenant=TENANT_ID, expiresIn=3600)
+    TOKEN = DataLakeCredential(
+        dict(
```

Comment:
So a dict literal for this would be:
```python
{
  'access': str(base64.b64encode(os.urandom(1420)),
  'refresh': str(base64.b64encode(os.urandom(718))),
  ...
}
```